doc/lua_api.txt

```diff
@@ -235,6 +236,56 @@ you have an item called `epiclylongmodname:stuff`, you could do
 
 and be able to use `/giveme stuff`.
 
+Mapgen aliases
```

Comment:
`###` to be a subheading

Reply:
I think it's important enough to have it's own section, makes it visually easier to find too.